common/workunit/workunit.hpp

```diff
@@ -1089,13 +1097,18 @@ class WORKUNIT_API WuScopeFilter
     const ScopeFilter & queryIterFilter() const;
     bool isOptimized() const { return optimized; }
     bool onlyIncludeScopes() const { return (properties & ~PTscope) == 0; }
+    WuScopeSourceFlags querySources() const { return sourceFlags; }
+    unsigned queryMinVersion() const { return minVersion; }
+    bool outputDefined() const { return properties != PTnone; }
 
 protected:
     void addRequiredStat(const char * filter);
+    void checkModifiable() { if (unlikely(optimized)) reportModifyTooLate(); }
     bool matchOnly(StatisticScopeType scopeType) const;
+    void reportModifyTooLate();
 
     //MORE: Make the following protected/private
-public:
+protected:
```

Comment:
This one is unnecessary as this follows through from protected section defined in line 1104.

Reply:
ok.  I will also remove the comment.

Reply:
Actually, I prefer to leave that it, since it marks the start of the data section.  But have removed the comment.